sql/moz-fx-data-shared-prod/app_store_v2_syndicate/dataset_metadata.yaml

```diff
@@ -0,0 +1,26 @@
+friendly_name: app_store_syndicate
```

Comment:
Should this dataset have Firefox in the name, since it's specifically Firefox and not all Mozilla apps?

Reply:
Actually, looking in BigQuery I now see the earlier `moz-fx-data-bq-fivetran.firefox_app_store` dataset was syndicated to an `app_store_syndicate` dataset, so this is in keeping with that precedent, which I guess makes sense for consistency.  So I retract my suggestion.